### ISSUE TITLE
Finishing Scenario #15 results in looping javascript error preventing using app

### ISSUE THREAD

**Issue by Zxurian** [Lurkars/gloomhavensecretariat#561]
### Describe the bug

Running 4 characters in Scenario 15. Upon hitting "Success", only part of the Scenario Summary shows up. Javascript console shows permanent looping throwing error..

![image](https://github.com/Lurkars/gloomhavensecretariat/assets/573371/5a740c37-7da7-45af-8ad6-2c83054b76c8)

I was not able to use the app at all after this occured. I had to open console, break one execution before the error being thrown, then update
```javascript
i.collectiveResources[0] = 0
i.collectiveResources[1] = 0
i.collectiveResources[2] = 0
i.collectiveResources[3] = 0
```
then was at least able to get back into app and cancel scenario, allowing future usage

### Game Enviroment

v0.95.16
Scenario 15

~~attach data export to replicate issue.~~

### To Reproduce

1. Import data
2. Set Scenario 15
3. Click "Ancien tSpire" at bottom
4. Click "Success"

### Expected behavior

scenario should display all rewards

### Gloomhaven Secretariat Version

0.95.16

### Used Browser

Chrome

### OS

Windows 11

**Comment by Lurkars** [comment]
Thanks for the report, should be fixed in [v0.95.17](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.95.17). Happy if you can verify and close issue. FYI: I removed the Data Dump from your comment, because the file contains your game code!

**Comment by Zxurian** [comment]
confirmed working. Just loaded it up and was able to complete it.
Thank you for removing game dump & quick turn around.
